### PR TITLE
BI-1298-Create-rspec-test-for-list_cps_to_commit

### PR DESCRIPTION
**Pull request by nguyentranlogigear**
Create rspec test for list_cps_to_commit method with 2 cases:
-1 returns correct commit cherry picked when existing repo
-2 returns correct commit cherry picked when non-existing repo

### CODE REVIEW

**Review by thanhnguyen19 — COMMENTED** [review]
Please check case: <code>response.reply(t('git.fetching'))</code> and <code>response.reply(t('git.cloning'))</code>

File: workspace/spec/lita/handlers/teamcity_spec_anki.rb [hunk 1]
```diff
@@ -471,4 +471,41 @@
 
   end
 
+  describe '#list_cps_to_commit' do
+    repo_name = "build-tools"
+    repos_dir = File.join(Dir.home, 'repos')
+    repo_uri  = "git@github.com:anki/#{repo_name}.git"
```
Comment (nguyentunglam912) [inline workspace/spec/lita/handlers/teamcity_spec_anki.rb]: Replace company-specific-info

Comment (nguyentranlogigear) [reply workspace/spec/lita/handlers/teamcity_spec_anki.rb]: I will update all company-specific-info after [PR #BI-1413](https://ankiinc.atlassian.net/browse/BI-1413) is approved and merge

**Comment by hiepky** [comment]
LGTM

**Pull request merged**